src/Sulu/Bundle/TrashBundle/Infrastructure/Doctrine/Repository/TrashItemRepository.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TrashBundle\Infrastructure\Doctrine\Repository;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityRepository;
+use Sulu\Bundle\TrashBundle\Domain\Exception\TrashItemNotFoundException;
+use Sulu\Bundle\TrashBundle\Domain\Model\TrashItemInterface;
+use Sulu\Bundle\TrashBundle\Domain\Repository\TrashItemRepositoryInterface;
+
+class TrashItemRepository implements TrashItemRepositoryInterface
+{
+    /**
+     * @var EntityManager
```

Comment:
Should be `EntityManagerInterface`
```suggestion
     * @var EntityManagerInterface
```